rust/fair-launch/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "fair-launch"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "fair_launch"
+
+[features]
+no-entrypoint = []
+no-idl = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+anchor-lang = "0.14.0"
+anchor-spl = "0.14.0"
```

Comment:
you may want to update anchor to 0.15

Reply:
how the f is he releasing so damned fast